kratos/python/add_model_part_to_python.cpp

```diff
@@ -938,6 +935,9 @@ void AddModelPartToPython(pybind11::module& m)
         .def("GetNonHistoricalVariablesNames", [](ModelPart& rModelPart, ModelPart::ConditionsContainerType& rContainer, bool doFullSearch=false) -> std::unordered_set<std::string> {
             return GetNonHistoricalVariablesNames(rModelPart, rContainer, doFullSearch);
         })
+        .def("HasMasterSlaveConstraint", [](ModelPart& rModelPart, ModelPart::IndexType MasterSlaveConstraintId) -> bool {
+            return rModelPart.HasMasterSlaveConstraint(MasterSlaveConstraintId);
+        })
```

Comment:
Can't we do it without the lambda in this case?

Reply:
Argggg... :rage:

Reply:
Mesh index delenda est